libs/auth-api-lib/src/lib/entities/dto/delegation.dto.ts

```diff
@@ -0,0 +1,33 @@
+import { ApiProperty } from '@nestjs/swagger'
+import {
+  IsBoolean,
+  IsString,
+  IsOptional,
+  IsNumber,
+  IsDateString,
+} from 'class-validator'
+// For use with deleting Inserting and updating Delegation
+export class DelegationDTO {
+  @IsString()
+  @ApiProperty()
+  fromNationalId!: string
+  @IsString()
+  @ApiProperty()
+  fromDisplayName!: string
+  @IsBoolean()
+  @ApiProperty()
+  isFromCompany!: boolean
```

Comment:
What is this for?

Reply:
Good point, not sure what i was thinking when i added this to our table. We should drop it.